includes/class-wc-payment-token-swedbank-pay.php

```diff
@@ -179,7 +183,11 @@ public static function wc_account_payment_methods_column_method( $method ) {
 		// @see woocommerce/myaccount/payment-methods.php
 		if ( ! empty( $method['method']['last4'] ) ) {
 			/* translators: 1: credit card type 2: last 4 digits */
-			echo sprintf( __( '%1$s ending in %2$s', 'woocommerce' ), esc_html( wc_get_credit_card_type_label( $method['method']['brand'] ) ), esc_html( $method['method']['last4'] ) );
+			echo sprintf(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of echo language construct is discouraged.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-woocommerce-payments/pullRequest?prid=5340032)